api/api_test.go

```diff
@@ -252,19 +252,6 @@ func TestAdminRedirect(t *testing.T) {
 	}
 }
 
-func TestPostPurchaseIndex(t *testing.T) {
```

Comment:
Can we put back this test? It's still useful to assert that we get an html response for the Post Purchase index page. We don't need to check what's in the response anymore but we should still assert that it succeeds

Reply:
That's fair.